services/checks/registry.go

```diff
@@ -91,8 +91,8 @@ func (r *Registry) CreateAlert(id string, labels, annotations map[string]string,
 	}
 }
 
-// RemovePrefix removes all alerts with given ID prefix except a given list of IDs.
-func (r *Registry) RemovePrefix(prefix string, keepIDs map[string]struct{}) {
+// removePrefix removes all alerts with given ID prefix except a given list of IDs.
+func (r *registry) removePrefix(prefix string, keepIDs map[string]struct{}) {
```

Comment:
Looks like that function can be refactored. It will process only STT alerts now, so `prefix` always the same. And probably we can wipe all alerts from registry each iteration and add new as there are no alerts form other components in that registry.

Reply:
so I guess we should call this something like `flushAlerts` now? @artemgavrilov

Reply:
I'm not sure, `flush` typically means "send everything from buffer" or something like that. Here we clean the registry. 

I think we can call that method `clean()` and invoke it before https://github.com/percona/pmm-managed/blob/cd1aa44a579ed32a25c136701db08384eb94fc39/services/checks/checks.go#L342-L346

So `clean()` will remove all existing alerts from the registry, and then we will fill the registry with newly received alerts from STT.

Reply:
done @artemgavrilov